SgfDevs/Web.config

```diff
@@ -233,10 +233,10 @@
     <caching configSource="config\imageprocessor\cache.config" />
     <processing configSource="config\imageprocessor\processing.config" />
   </imageProcessor>
-  <system.codedom>
+  <!--<system.codedom>

```

Comment:
@mykebates any reason not to just completely remove this portion?

Reply:
I left some pretty lengthy details in the actual commit. But tldr; just not 100% certain on the risks of lifting it out entirely(removing the package/config/etc) and wanted to keep it easy as possible to quickly add back in or test adding back in wherever needed. Eventually I am hoping we can totally kill it but didn't want to until we were sure early setup was working on a few different machines.